features/machine/machine_healthcheck.feature

```diff
@@ -10,7 +10,7 @@ Feature: MachineHealthCheck Test Scenarios
     And I use the "openshift-machine-api" project
 
     Given I store the number of machines in the :num_to_restore clipboard
-    And admin ensures machine number is restored to "<%= cb.num_to_restore %>" after scenario
+    And admin ensures machine number is restored to after scenario
```

Comment:
remove "to", the step is "admin ensures machine number is restored after scenario"